server/templates/choropleth.html

```diff
@@ -0,0 +1,40 @@
+{#
+    Copyright 2020 Google LLC
+   
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+   
+         http://www.apache.org/licenses/LICENSE-2.0
+   
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+   #}
+   
+   {% extends 'base.html' %}
+   
+   {% set is_hide_full_footer = true %}
+   {% set subpage_title = "Choropleth" %}
+   {% set title = "Choropleth tool" %}
+   
+   {% block head %}
+   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" >
+   <link rel="stylesheet" href={{url_for('static', filename='css/choropleth.min.css')}} >
+   {% endblock %}
+   
+   {% block content %}
+   <div id="main-pane"></div>
+   {% endblock %}
+   
+   {% block footer %}
+     <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/4.9.1/d3.min.js" defer></script>
+     <script src="https://d3js.org/d3-geo.v1.min.js" defer></script>
+     <script src={{url_for('static', filename='choropleth.js', t=config['GAE_VERSION'])}}></script>
+   {% endblock %}
+   
+   {% block subfooter_extra %}
```

Comment:
This probably isn't required yet? Remove for now until we add this feature.

Reply:
Removed.